### PR TITLE
Upgrade ibm provider for token expire issue

### LINKED ISSUE #370: Update required IBM Cloud Providers to 1.39.0
Now we often got OCP installation fail due to token expired error when call to IBM cloud. This token expiration bug is fixed in TF IBM cloud provider release 1.39.0.

### PR DESCRIPTION
**Pull request by yussufsh**
Fixes #370

Signed-off-by: Yussuf Shaikh <yussuf.shaikh@ibm.com>

### CODE REVIEW

**Review by sudeeshjohn — APPROVED** [review]
/lgtm

File: modules/1_prepare/versions.tf [hunk 1]
```diff
@@ -22,7 +22,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "1.36.0"
+      version = "1.39.0"
```
[the remark this answers is not in the document]
Comment (yussufsh) [reply modules/1_prepare/versions.tf:25]: As discussed lets lock it at a particular version where we know most of the power code works fine.

**Pull request merged**